app/frontend/stores/user-store.ts

```diff
@@ -31,7 +35,7 @@ export const UserStoreModel = types
   }))
   .actions((self) => ({
     setUsers(users) {
-      R.forEach((u) => self.usersMap.put(u), users)
+      R.map((u) => self.usersMap.put(u), users)
```

Comment:
It's better to use forEach over map in this case. Map creates unnecessary memory which is not being used anywhere

Reply:
roger